samples/ribbon/ribbondemo.cpp

```diff
@@ -195,6 +195,10 @@ bool MyApp::OnInit()
     if(!wxApp::OnInit())
         return false;
 
+#ifdef __WXMSW__
```

Comment:
I'd rather omit this, we have a universal way of enabling dark mode for all samples by setting `wx_msw_dark_mode` environment variable and I'd like to have consistent behaviour for all of them.

Reply:
Done.